# List of joint configurations
# Array indices from 0 to max match joints from base joint to last joint
JointConfigMeasured[] joint_config_measured
